src/routes/MemberProfile/MemberProfile.js

```diff
@@ -131,6 +135,12 @@ export default class MemberProfile extends React.Component {
           </div>
           <SkillsToLearnSection personId={personId} editable={false} />
 
+          { memberships && memberships.length > 0 && <div styleName='profile-subhead'>Hylo Communities</div> }
+          { memberships && memberships.length > 0 && memberships.map((m, index) => <Membership key={m.id} index={index} membership={m} />) }
```

Comment:
i forget, does this do any checking on the back-end of whether the viewer is allowed to see this hylo community? im pretty sure we agreed that people can only see communities that the other person is a part of if it is public or if they too are a part of that community...

Reply:
There's logic in the front-end to prevent the viewer from seeing the community if they are not part of it. Currently that is also filtering out public communities, I'm not sure why that is happening. Also in testing this I recreated the bug where a user's role shows up as "moderator" when the viewer is a moderator of that community.

Reply:
Recap of our discussion in sprint planning - we agreed it works for the viewer to only see communities that they are also a member of (which is the functionality here). We should test whether the "moderator" bug appears in staging, as it could be a quirk of trying to log in as two different users locally.